index.html

```diff
@@ -157,34 +159,34 @@
 						---
 
 						### Head
-						Contains the page title and meta information about the page. Meta information is invisible to the user, but has many purposes, like providing information to search engines.
+						Contains the page title and meta information about the page. Meta information is mostly invisible to the user, but has many purposes, like providing information to search engines.
 
 						---
 
 						### Body
-						Contains the content of the page. Everything in the body is visible to the user.
+						Holds the visible content of the page. This is where you put anything you wish the user to see.
 
 						---
 
 						### Example
 						``` html
 						<!DOCTYPE html>
-							<html>
-							  	<head>
-							    	<title>Title of the page</title>
-							  	</head>
-							  	<body>
-							    	The page content here.
-								</body>
+						<html>
+							<head>
+								<title>Title of the page</title>
+							</head>
+							<body>
+								The page content here.
+							</body>
 						</html>
 						```
 
 						---
 
 						### Nesting
-						All elements “nest” inside one another. Nesting occurs when elements are inside other containing tags.
+						HTML is structured by nesting elements inside other elements. This is done by placing elements inside other containing tags.
 
-						For example, you would put the <p>inside of the <body> tags. The <p> is now nested inside the <body>
+						For example, you would put the `<p>`inside of the `<body>` tags. The `<p>` is now nested inside the `<body and is a "child" or a "descendent" of the body.
```

Comment:
Introduced the parent/child relationship here, since we need to know this later.

Reply:
body is missing an `>` tag